appveyor.yml

```diff
@@ -1,6 +1,8 @@
 # AppVeyor.com is a Continuous Integration service to build and run tests under Windows
 # see https://github.com/ogrisel/python-appveyor-demo/blob/master/appveyor.yml
 
+clone_depth: 50
```

Comment:
Shallow git clones will lead to wrong/misleading version numbers in test reports because our version number lookup depends on `git-describe`. I don't see how this change is needed for uploads to pypi. I really think this change should be reverted.